.github/workflows/unit-tests-helm.yaml

```diff
@@ -20,7 +20,8 @@ jobs:
     container:
       image: ghcr.io/gravitational/teleport-buildbox:teleport12
       env:
-        HELM_PLUGINS: /root/.local/share/helm/plugins
+        # TODO(hugoShaka) remove the '-new' prefix when updating to teleport13 buildbox
```

Comment:
In teleport13 we can probably just get rid of this environment variable, no?

Reply:
If we set it at the container level, yes

Reply:
It's not clear for me which user the CI is running as. If it's `root` root `ci` we should be able to remove it entierly. However the fact the variable was set previously hints this is not the case and someone had an issue with helm plugin not found.